lib/src/screens/metamap/metamap_controller.dart

```diff
@@ -15,21 +15,14 @@ class MetaMapController {
     {"name": "Otro"},
   ];
 
-  Future<bool> updateUser({required User user}) async {
+  Future<bool> updateUser({required String address, required String profession}) async {
```

Comment:
Acá entonces recibes el input

```suggestion
  Future<bool> updateUser(UpdateUserInput: updateUserInput) async {
```